release notes/v1.1.0.md

```diff
@@ -0,0 +1,113 @@
+k6 `v1.1.0` is here 🎉! This release includes:
+
+- New `count`, `nth`, `first` and `last` methods on the `Locator` API [#4797](https://github.com/grafana/k6/pull/4797), [#4825](https://github.com/grafana/k6/pull/4825)
+- The `k6/experimental/webcrypto` module is now globally available.
+- Group results in the `full` end-of-test summary are now sorted as in code and properly indented.
+
+## Breaking changes
+
+### Remove experimental `k6/experimental/webcrypto` module [#4851](https://github.com/grafana/k6/pull/4851)
+
+The `k6/experimental/webcrypto` module is now globally available, and the experimental import is no longer available.
```

Comment:
```suggestion
The [WebCrypto API](https://grafana.com/docs/k6/latest/javascript-api/crypto/) has been globally available since v1.0.0-rc1 (or v0.58.0), and now the  experimental import (`k6/experimental/webcrypto`) is no longer available.

The required change for users should be to just remove the `import` and the rest of the code should just work. 
```